### PR TITLE
Update easy-ssh.sh

### PR DESCRIPTION
**Pull request by sean-smith**
Update command to echo directly to the `~/.ssh/config` file, saving a step

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: 1.architectures/5.sagemaker-hyperpod/easy-ssh.sh [hunk 1]
```diff
@@ -61,9 +61,11 @@ echo -e "aws ssm start-session "${aws_cli_args[@]}" --target sagemaker-cluster:$
 
 echo -e "Add the following to your ~/.ssh/config to easily connect:"
 echo "
+cat <<EOF >> ~/.ssh/config
```
Comment (KeitaW) [inline 1.architectures/5.sagemaker-hyperpod/easy-ssh.sh:64]: The script may require multiple executions, such as when a user forgets to add a key to the server. Currently, each run of the script appends the host configuration, which is not optimal. Introducing an optional argument to toggle this feature could help prevent unnecessary repetitions.

Comment (sean-smith) [reply 1.architectures/5.sagemaker-hyperpod/easy-ssh.sh:64]: This is just echo'd out and not actually written to the file.

Comment (KeitaW) [reply 1.architectures/5.sagemaker-hyperpod/easy-ssh.sh:64]: Sorry you are right.

**Review by KeitaW — APPROVED** [review]
LGTM

**Pull request merged**